internals/plan/plan.go

```diff
@@ -776,17 +731,24 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 			if _, ok := combined.Services[serviceName]; ok {
 				continue
 			}
-			return nil, &FormatError{
-				Message: fmt.Sprintf(`log target %q specifies unknown service %q`,
-					target.Name, serviceName),
-			}
+			errorMessages = append(errorMessages, fmt.Sprintf(`log target %q specifies unknown service %q`,
+				target.Name, serviceName))
+
 		}
 
 		if target.Location == "" {
-			return nil, &FormatError{
-				Message: fmt.Sprintf(`plan must define "location" for log target %q`, name),
-			}
+			errorMessages = append(errorMessages, fmt.Sprintf(`plan must define "location" for log target %q`, name))
+
+		}
+	}
+
+	// Ensure combined layers don't have errors
+	if len(errorMessages) > 0 {
+		fullErrorMessage := "\ninvalid plan:\n- " + strings.Join(errorMessages, "\n- ")
```

Comment:
How about we keep these error messages the same when `len(errorMessages) == 1`, and note the multiple errors when >1. And let's drop the leading `\n`. Maybe:

```go
if len(errorMessages) == 1 {
    return nil, &FormatError{Message: errorMessages[0]}
}
if len(errorMessages) > 0 {
    message := "multiple errors validating plan:\n- " + strings.Join(errorMessages, "\n- ")
    return nil, &FormatError{Message: message}
}
```